### PR TITLE
Locale update, pull and fixes

### PR DESCRIPTION
**Pull request by lzap**

### CODE REVIEW

**Comment by lzap** [comment]
Although there are few more strings in the upcoming bugs let's just merge this @orrabin so translators have fresh meat for now.

**Comment by lzap** [comment]
Can you guys take a look? Thanks @GregSutcliffe or @domcleal :-D

File: locale/foreman_discovery.pot [hunk 1]
```diff
@@ -77,19 +147,46 @@ msgstr ""
...
 msgstr ""
 
 msgid "Failed to refresh facts for %s"
 msgstr ""
 
-msgid "Imported Host::Discovered"
+msgid "Host"
+msgstr ""
+
+msgid "Host %s was provisioned with rule %s"
```
Comment (domcleal) [inline locale/foreman_discovery.pot]: Shouldn't use two %s within one string - I suggest you fix the offending code and re-extract.

**Comment by domcleal** [comment]
Looks fine apart from one string.  I'd encourage you to avoid string changes in bug fixes though.

**Comment by lzap** [comment]
Actually we have't extracted yet for the RC1, this is the first time :-P

Will re-extract, I see one more bug with a string change, will do that first.

**Comment by lzap** [comment]
Pushed string extraction. I also pulled new translations but there are none.

**Comment by lzap** [comment]
@GregSutcliffe can you take a look please? RC2 is scheduled for tomorrow.

**Comment by domcleal** [comment]
There aren't going to be any translations until you push an updated POT to the repo.. it really needs to be done earlier!

**Comment by orrabin** [comment]
merged as ab4c89e22433fbaaf95d035b87659cd20fc48268, thanks @lzap!

**Pull request closed without merging**